db/models/seeders.js

```diff
@@ -0,0 +1,165 @@
+const { query } = require('../index');
+
+module.exports = {
+  seedStudents: (students) => {
+    return new Promise((resolve, reject) => {
+      students.forEach(async(student, index) => {
+        let q = `
```

Comment:
pet peeve:  single letter variables.  I make an exception for CS convention like looping vars, events in error or event handling.

Would love to define a non-single-character convention for us to use here.